config/config.go

```diff
@@ -205,7 +205,11 @@ func SetStartupParameter(startupFlag *command.Args, cfg *Config) error {
 	cfg.GCPeriod = d
 
 	cfg.Address = startupFlag.Address
-	cfg.APISocket = startupFlag.APISocket
+
+	if startupFlag.APISocket == "" {
+		cfg.APISocket = path.Join(cfg.RootDir, "api.sock")
```

Comment:
It seems we should allow setting the `APISocket` to empty.

Reply:
Do you mean completely shutdown api server?

Reply:
I mean how do users set it to empty?